man/rpm-ostree.xml

```diff
@@ -522,11 +527,66 @@ Boston, MA 02111-1307, USA.
         <listitem>
           <para>
             Without options, display current default kernel arguments. Modify
-            arguments using <option>--append</option>,
-            <option>--replace</option>, <option>--delete</option>, or
-            <option>--editor</option>. This will create a new deployment with
-            the modified kernel arguments. Previous deployments are never
-            changed.
+            arguments using the following parameters which will create a new 
+            deployment with the modified kernel arguments. Previous deployments
+            are never changed.
+          </para>
+
+          <para>
+            <command>
+              --editor
+            </command>
+              to use an editor to modify the kernel arguments.
+          </para>
+
+          <para>
+            <command>
+              --append
+            </command>
+              to append a kernel argument. For example, 
+              <command>--append=panic=1</command>.
+          </para>
+
+          <para>
+            <command>
+              --append-if-missing
+            </command>
+              to append a kernel argument if it is not present.
```

Comment:
I think this has implications on the return code for those additional actions. Maybe we should mention that?

Reply:
Looking at https://github.com/coreos/rpm-ostree/pull/2713/files is not clear to me where the return changes if the arg is missing or not. I bet I am missing something obvious.

Reply:
Whoops, sorry for the confusion.